packages/plugin-eslint/src/lib/runner/transform.ts

```diff
@@ -66,7 +66,7 @@ function convertIssue(issue: LintIssue): Issue {
     source: {
       file: issue.filePath,
       position: {
-        startLine: issue.line,
+        ...(issue.line > 0 && { startLine: issue.line }),
```

Comment:
Please change this so `position` isn't set if we don't have a valid start line. I.e. something like `...issue.line > 0 && { position: { startLine: issue.line, ... } }` :pray: